panther_analysis_tool/main.py

```diff
@@ -223,8 +227,8 @@ def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any,
                     logging.debug("Skipping path %s", relative_path)
                     continue
             for filename in sorted(file_list):
-                # Skip hidden files
-                if filename.startswith("."):
+                # Skip hidden files and files explicitly ignored
+                if filename.startswith(".") or filename in ignored_files:
```

Comment:
Hmm are you sure about this? Wouldn't this result in ignoring every file `test.yml` regardless of the directory? I think it is safer to require the relative path from the location where the executable runs. So if a user ignores `test.yml`, they actually mean `ignore "$PWD/test.yml"`.

Also I'm not sure where we stand on logging and exceptions, but consider:
- printing a warning log when encountering an ignored path
- raising an error if you don't encounter it, since that's an indication of a problem as well (mistyped path perhaps)

Reply:
Agreed on both counts. I thought an INFO log might be more appropriate though. 

I intentionally did not raise an error if its not found because the next step on this will be a config file to read in args and I didn't want peoples CI jobs to fail because they removed a file and forgot to update the config file. All this config file is coming on another PR

Reply:
It would be neat if we could support fnmatch